Src/WitsmlExplorer.Frontend/components/Sidebar/ServerManager.tsx

```diff
@@ -132,6 +143,7 @@ const ServerManager = (): React.ReactElement => {
         <Select labelId="servers-label" value={selectedServer?.id ?? ""} onOpen={() => setIsOpen(true)} onClose={() => setIsOpen(false)}>
           {servers
             .sort((a, b) => a.name.localeCompare(b.name))
+            .filter((server: Server) => !msalEnabled || (msalEnabled && server.securityscheme === securityScheme.OAuth2 && userAppRoles.includes(server.role)))
```

Comment:
When editing a server, the role is said to be space delimited (for example "admin developer") but here it is checked like userAppRoles.includes(server.role), instead of comparing contents of two arrays?